pipelines.adoc

```diff
@@ -1,266 +1,189 @@
-leverages buildah, also requires a volume mount. The minimum required
-volume size is 5 Gi.
-
-If your cluster is running in a public cloud, dynamic volume
-provisioning is the easier and preferred option. Please refer to the
-following doc that describes how to point to the storage class using a
-config map.
-https://github.com/tektoncd/pipeline/blob/master/docs/install.md#how-are-resources-shared-between-tasks
-
-Note: If this is not configured and based on how your cloud provider’s
-default storage class is configred, your pipelines might create a new
-volume for each run which will increase your pipeline run execution
-time.
-
-Alternatively, if you are using an unmanaged cluster you can setup a
-persistent volume to be used by the pipeline. A simple example pv
-definition is provided. Update path and other values in pv.yaml to suit
-your requirements. For example, NFS is the preferred configuration in a
-production environment verus using local storage as shown in the
-example.
-
-Login to your cluster. For example for OKD,
-
-....
+=== Prerequisites
+
+. link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation] must be installed on Red Hat Origin Community Distribution of Kubernetes (OKD) or the OpenShift Container Platform (OCP) cluster.
+
+. link:https://github.com/tektoncd/dashboard[Tekton Dashboard] is installed by default with Kabanero Foundation. To find the Tekton Dashboard URL, login to your OKD cluster and run `+oc get routes+` or check in the OKD console.
+
+
+
+==== Set up dynamic volume provisioning or a persistent volume.
+
+Tekton pipelines require a configured volume that is used by the framework to share data across tasks. The build task, which uses Buildah, also requires a volume mount. The minimum required volume size is five Gi.
+
+If your cluster is running in a public cloud, dynamic volume provisioning is the easier and preferred option. For more information, see link:https://github.com/tektoncd/pipeline/blob/master/docs/install.md#how-are-resources-shared-between-tasks[How are resources shared between tasks].
+
+Your persistent volume must be configured based on how your cloud provider’s default storage class is configured. Otherwise, your pipelines might create a new volume for each run, which increases your pipeline run execution time.
+
+Alternatively, if you are using an unmanaged cluster, you can set up a persistent volume to be used by the pipeline. A simple example pv definition is provided in the following example. Update the path and other values in your `pv.yaml` file to suit your requirements. For instance, NFS is the preferred configuration in a production environment, rather than using local storage, as shown in the example.
+
+. Log in to your cluster. For example, for OKD:
+
+```
 oc login <master node IP>:8443
-....
+```
 
-Clone the pv.yaml in this repo and apply it.
+. Clone the `pv.yaml` file in this repo and apply it.
```

Comment:
We should say get the example pv.yaml provided in the kabanero-pipelines repo over here:  https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/common/pv.yaml

Reply:
Should this be a distinct step (`Clone the kabanero-pipelines repo..`) before cloning the pv.yaml?

Reply:
Think that makes sense.  We do that in some of the later sections.